src/routes/PublicLayoutRouter/PublicLayoutRouter.js

```diff
@@ -64,12 +65,50 @@ export function PublicGroupDetail (props) {
   }
 
   return (
-    <div styleName='center-column non-map-view' id={CENTER_COLUMN_ID}>
-      <GroupDetail {...props} />
-    </div>
+    <>
+      <div styleName='center-column non-map-view' id={CENTER_COLUMN_ID}>
+        <GroupDetail {...props} />
+      </div>
+      <div />
+      <Route
+        path={`(.*)/${POST_DETAIL_MATCH}`}
```

Comment:
we might want to match something after the post details too? like if someone copies an edit link?

Reply:
The path match is not `exact`, so it should catch anything after this match as well